machines/backupAndRestore/backupAndRestore.ts

```diff
@@ -0,0 +1,214 @@
+import {EventFrom, StateFrom} from 'xstate';
+import {createModel} from 'xstate/lib/model';
+import {AppServices} from '../../shared/GlobalContext';
+import Cloud, {
+  ProfileInfo,
+  SignInResult,
+  isSignedInResult,
+} from '../../shared/googleCloudUtils';
+import NetInfo from '@react-native-community/netinfo';
+import {ErrorMessage} from '../../shared/openId4VCI/Utils';
+import {NETWORK_REQUEST_FAILED} from '../../shared/constants';
+
+const model = createModel(
+  {
+    isLoading: false as boolean,
+    profileInfo: undefined as ProfileInfo | undefined,
+    errorMessage: '' as string,
+  },
+  {
+    events: {
+      HANDLE_BACKUP_AND_RESTORE: () => ({}),
+      PROCEED: () => ({}),
+      GO_BACK: () => ({}),
+      TRY_AGAIN: () => ({}),
+      DISMISS: () => ({}),
+    },
+  },
+);
+
+export const BackupAndRestoreEvents = model.events;
+
+export const backupAndRestoreMachine = model.createMachine(
```

Comment:
Does this have any functionality of backup other that restore?

Reply:
No we dont have any backup / restore functionality here. This machine deals with 
-> cloud account selection
-> User infor of associated cloud account
-> open Backup and restore screen

Reply:
Lets get this renamed then.

Reply:
Renamed to backupAndRestoreSetup